### PR TITLE
refactor(*) integrate with `aws-lambda`

### PR DESCRIPTION
**Pull request by ghost**
This commit integrates the `liamp` plugin with the `aws-lambda` plugin.

The `liamp` plugin should be renamed back to `aws-lambda`, stay in its
own external repo, and then the upstream changes would be to remove
Lambda there, and update the rockspec to include this external one.

### CODE REVIEW

File: kong-plugin-aws-lambda-0.2.0-1.rockspec [hunk 1]
```diff
@@ -11,7 +11,7 @@ local pluginName = package:match("^kong%-plugin%-(.+)$")  -- "myPlugin"
 
 supported_platforms = {"linux", "macosx"}
 source = {
-  url = "http://github.com/Tieske/kong-plugin-liamp.git",
+  url = "http://github.com/Tieske/kong-plugin-aws-lambda.git",
```
Comment (Tieske) [inline kong-plugin-aws-lambda-0.2.0-1.rockspec]: Tieske -> kong

**Review by Tieske — CHANGES_REQUESTED** [review]
repo name needs to be updated (see comment), also the version needs to be bumped to 3.0.0 to supersede the current 2.0.0 in Kong open source

**Comment by Tieske** [comment]
note: version bump should also reflect in the rockspec filename and contents

**Comment by ghost** [comment]
Looking at the build logs [1] Kong seems to prioritize `aws-lambda` from the core repository instead of this custom one. So In order for the tests to pass I believe we'll need to remove `aws-lambda` from core first.

[1] https://travis-ci.com/Kong/kong-plugin-liamp/jobs/240845318#L728-L729

**Pull request merged**